lib/runner/extensions/event.command.js

```diff
@@ -28,6 +33,56 @@ var _ = require('lodash'),
             error.name = ASSERTION_FAILURE;
         }
         return error ? serialisedError(error, true) : undefined;
+    },
+
+    /**
+     * Removes files in Request body if any.
+     *
+     * @param {Request~definition} request Request JSON representation to be sanitized
+     * @param {Function} callback function invoked with error and sanitizedRequest.
+     *                            sanitizedRequest is `null` if no files are detected.
+     */
+    sanitizeFiles = function (request, callback) {
```

Comment:
I think callback should always get the request from parameter so that it doesn't have to refer to the outer closure. ... as such the signature of callback could be `error:Error, request:Request, affectedEntries:Array.<src:String>` - in this way, we would know that nothing was affected if `_.isEmpty(affectedEntries) = true`.